src/App.tsx

```diff
@@ -26,11 +25,11 @@ const MattapanMappingRoutes = () => (
     <Route path='*'><NotFound /></Route>
   </Switch>
 )
+
 export default function App() {
   return (
     <ThemeProvider theme={Theme}>
       <Router>
-        <Header />
```

Comment:
I put the header in here because it is shared by all pages in the application. I think if you move this as part of each page, the header ends up re-rendering (unmount/remount) when you click on a new page?

Reply:
As I type that and tested myself, it seems (to my eyes at least) the app currently does full page refresh anyway and some stackoverflow suggestions say to use the `Link` component from react-router instead of anchor tag - see https://github.com/codeforboston/mattapan-mapping/issues/82